flow-typed/npm/babel-preset-env_vx.x.x.js

```diff
@@ -0,0 +1,88 @@
+// flow-typed signature: 84e004258eec612c8ccb08e44dce7287
```

Comment:
I don't really know if we need typedefs for most of the devDependencies, especially generated `any` stubs.